### PR TITLE
NT14-WaveShot

### PR DESCRIPTION
**Pull request by Lokey82**
## About the PR
Adds the NT14-WaveShot, The long-awaited reporter ship based on the Goonstation Radio Lab. This ship has been fit to serve the purpose of a news station with a reporter's office centerpiece that contains even two portable entertainment cameras and a mass-media console. This ship currently goes for a nice 40 thousand spessos and is well worth the price with its many luxurious amenities including but not limited to many vending machines, A captain's quarters, and of course the reporter's office plus a waiting room.

I hope that you get to see this ship soon amongst the cosmos at a Frontier outpost near you!

## Why / Balance
I felt like making a ship and noticed there were no reporter ships.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

## Breaking changes

**Changelog**

:cl:
- add: CentCom has become tired of supplying Esquires with mass-media consoles, so they have contracted the help of a new ship engineer. Please welcome the NT14-WaveShot, A new ship for reporters!

### CODE REVIEW

File: Resources/Maps/Shuttles/NT14-WaveShot.PNG
Comment (FoxxoTrystan) [inline Resources/Maps/Shuttles/NT14-WaveShot.PNG:1]: - This file should not be here.
Merge request that are open should be the final product.

**Review by FoxxoTrystan — APPROVED** [review]
- File name should be "waveshot" as NT14 will be auto-generated, best to keep lowcaps on files names.
- Uneeded PNG File (copy and paste image on your PR Info that works)
- Missing Shipyard file.
- Made on upstream? this already cause a ton of issues due to component and ent that are only on Frontier.
- Incorrect Fax machine, replace it with a Ship Fax Machine.
- No Jobs Spawns (Is it normal?)
- No ShipWarpPoint.
- No LateJoinPoint. (If there no jobs to add then i supose this is uneeded)
- Missing "Becomestation" component.
- Bridge airlock are command locked, nobody will be able to open those doors.

Suggestions:
- Missing Piping color, use colornetwork.
- Avoid adding a Mass Scanner Console on ships that do not need it. (should be kept for bounty hunting ships, expeditions, ect...)
- Internal doors should be normal solid walls and not reinforced walls, reinforced walls should be kept for the external hull.
- 3 APC connect the same LV, this should be avoided.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
- File name should be "waveshot" as NT14 will be auto-generated, best to keep lowcaps on files names.
- Uneeded PNG File (copy and paste image on your PR Info that works)
- Missing Shipyard file.
- Made on upstream? this already cause a ton of issues due to component and ent that are only on Frontier.
- Incorrect Fax machine, replace it with a Ship Fax Machine.
- No Jobs Spawns (Is it normal?)
- No ShipWarpPoint.
- No LateJoinPoint. (If there no jobs to add then i supose this is uneeded)
- Missing "Becomestation" component.
- Bridge airlock are command locked, nobody will be able to open those doors.

Suggestions:
- Missing Piping color, use colornetwork.
- Avoid adding a Mass Scanner Console on ships that do not need it. (should be kept for bounty hunting ships, expeditions, ect...)
- Internal doors should be normal solid walls and not reinforced walls, reinforced walls should be kept for the external hull.
- 3 APC connect the same LV, this should be avoided.

**Comment by Lokey82** [comment]
It is with great regret that I announce the closure of this PR, Due to multiple issues including the ones pointed out. This ship cannot be added to the game. I found solutions to these problems but unfortunately could not apply them. My inability to take the development of this project further and allow its implementation into the game is why I will be closing this PR as the ship would only sit uselessly in the mass of the other PRs here. If I can apply the solutions to the problems occurring I may return to this project in the future. But for now, I bid you all farewell and thank you for the opportunity to attempt to work on this amazing game.

**Comment by Cheackraze** [comment]
what issues did you have? Dont let our slower dev process deter you, and I appologize for not getting to it sooner, but I have been typically doing ship reviews just once a week.  I would love to see this in game still, get at me on discord so I can help you get it sorted

**Comment by Lokey82** [comment]
Please disregard the closure of this pull request as stated in my previous comment. Thanks to the help of the wonderful developers of this game all issues have been solved and progress has resumed.

File: Resources/Maps/Shuttles/NT14-WaveShot.PNG
Comment (Lokey82) [reply Resources/Maps/Shuttles/NT14-WaveShot.PNG:1]: All of your issues mentioned have been solved in the latest commit, Thank you for your feedback.

File: RobustToolbox
Comment (Lokey82) [inline RobustToolbox:1]: Honestly, I have no clue why this one is here. I think I fucked up when committing.

Comment (Lokey82) [reply RobustToolbox:1]: Feel free to direct me to a fix as I got no clue what to do.

Comment (Cheackraze) [reply RobustToolbox:1]: looks like it slipped in by mistake. you can take your saved map file and revert this commit, re-apply the map file and make a new commit. If you have more troubles, I can help fix it for you in a day or so

**Comment by Cheackraze** [comment]
I added the shipyard prototype and finished up the remaining issues of the components and locked doors, it should be gtg

**Comment by Lokey82** [comment]
Ah, I just pushed a commit for the shipyard file and all that stuff. I guess git didn't email me that you did that.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**